src/aleph_vrf/coordinator/vrf.py

```diff
@@ -66,6 +62,19 @@ async def post_executor_api_request(url: str, model: Type[M]) -> M:
             return model.parse_obj(response["data"])
 
 
+async def prepare_executor_api_request(url: str) -> bool:
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url, timeout=120) as resp:
+            if resp.status != 200:
+                raise ExecutorHttpError(
+                    url=url, status_code=resp.status, response_text=await resp.text()
+                )
+
+            response = await resp.json()
+
+            return response["name"] == "vrf_generate_api"
```

Comment:
Why not use `resp.raise_for_status()` ?
https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.raise_for_status

```suggestion
            try:
                resp.raise_for_status()
                response = await resp.json()
                return response["name"] == "vrf_generate_api"
            except aiohttp.ClientResponseError as error:
                raise ExecutorHttpError(
                    url=url, status_code=resp.status, response_text=await resp.text()
                ) from error 
```